### ISSUE TITLE
Create an application summary view

### ISSUE THREAD

**Issue by danblah** [OpenTechFund/hypha#2214]
**Use-case**
We need a view we can send to staff and reviewer groups (via a URL) that displays an arbitrary group of proposals and a manager written summary.

Right now we manually create this in a google/notion doc or email and send to staff and managers. To reduce sensitive information in an email, we would like to stop having to do this. 

**Add summary to applications**
* On the application page (/apply/submissions/555/), on hover of the space between the title and requested funding, offer an "Add summary" option.
* On click, expand the page and expose a rich-text box (w/o reloading, no pop-up), with save and discard button, with a 500char limit counter.
* On save, display text in the same space.
* On hover of existing text, offer the ability to edit the text
* Discard on existing text deletes any existing text and saves as blank
* Ability to do this in the react app as well
* Viewable by all with access to the application, editable only by staff

**Grouping of applications**
* Ability to create a list of applications using any available filters in the apply/submissions/all or react app
* Click a new icon where the other two view changes are that takes selected applications and displays the list in the below format
* Ability to do this from react and non-react interfaces
* Ability to share a direct link to this view to users in role Staff and Reviewer

**List of applications**
_Application 1 (link to application)_
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam sit amet quam porttitor, viverra eros vel, auctor odio. Ut venenatis sem nec velit lobortis, sit amet sagittis est condimentum. Curabitur in dolor risus. Integer quis leo quis metus viverra vehicula. Suspendisse id dolor facilisis, tincidunt libero ut, bibendum elit. Donec eu posuere lectus. In interdum ultrices eros id eleifend. Suspendisse potenti. Aliquam purus nibh, molestie at nisl vitae, accumsan rhoncus tortor. Suspendisse ac.

_Application 2 without summary (link to application_

_Application 3 with summary (link to application)_
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam sit amet quam porttitor, viverra eros vel, auctor odio. Ut venenatis sem nec velit lobortis, sit amet sagittis est condimentum. Curabitur in dolor risus. Integer quis leo quis metus viverra vehicula. Suspendisse id dolor facilisis, tincidunt libero ut, bibendum elit. Donec eu posuere lectus. In interdum ultrices eros id eleifend. Suspendisse potenti. Aliquam purus nibh, molestie at nisl vitae, accumsan rhoncus tortor. Suspendisse ac.

etc etc

**Comment by Parbhat** [comment]
@vimal1083 Let's implement it first on React app. The details are almost complete. You can ask @danblah if have any doubts.

**Comment by vimal1083** [comment]
> Click a new icon where the other two view changes are that takes selected applications and displays the list in the below format

@danblah What to do you mean by `other two view changes`

**Comment by danblah** [comment]
Just tested the ability to add a summary in react on multiple apps , works great, awesome! The summary isn't visible on the application page yet so i can't test the view/edit permissions (ie, all can see only staff can edit).

cc @ei8fdb cause we talked a bit about this earlier today.

**Comment by ei8fdb** [comment]
@danblah Yup, I'm working on some design for this. Please gie me some time before what the user sees is impemented. @Parbhat if the design becomes a blocker in 2-3 days tag me here and I'll post it.

**Comment by Parbhat** [comment]
@ei8fdb An initial implementation is in PR #2249. You can check on the test environment. The latest changes are not deployed to test. @frjo We can re-deploy the PR to test? There is also some chat about the icon in PR https://github.com/OpenTechFund/hypha/issues/2248#issuecomment-789514883

**Comment by frjo** [comment]
Redeployed to test now.

**Comment by Parbhat** [comment]
@danblah Some updates have been made to PR. Need a few requirements feedback

1. The ability to add/edit summary is currently only in React app. Should we add the same feature to the normal Application page (example `/apply/submissions/19/`) in the first release?
2. The summary list of applications (`/apply/submissions/grouped-applications/?id=9,19`) is open to staff only. Should it be made accessible to reviewers? The react app is currently only for Staff so only staff can add/edit summary to applications. We can open the application summary list page to reviewers but adding/editing summary for reviewers have to be implemented on the normal Application page (example `/apply/submissions/19/`).

**Comment by ei8fdb** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Bernard to review the design in ticket: 2214](https://trello.com/c/Gsae0W5g/3-bernard-to-review-the-design-in-ticket-2214)

**Comment by ei8fdb** [comment]
I'm not able to test this. I don't know where to find this. @frjo @Parbhat ?

**Comment by Parbhat** [comment]
@ei8fdb Here is the screen recording from the test environment. It is currently a prototype and available only in 3-panel view (React app). Opening the issue so we can enhance it based on feedback.

https://user-images.githubusercontent.com/4817493/113407864-d79d8180-93cb-11eb-9f39-7e8b4627b964.mov

**Comment by Erioldoesdesign** [comment]
Q1 2022 Rebasing updates:
* Removed from milestones 'horizon 1' https://github.com/HyphaApp/hypha/milestone/24
* Added Needs design QA review label
* Recommend review in Q2 rebasing for priority level.
* Removed from dev backlog board: https://github.com/HyphaApp/hypha/projects/2 (for now)
* Added Theme:Admin label

* @frjo and @parbhat is this in a test/sandbox environment? It got merged here: https://github.com/HyphaApp/hypha/pull/2249 but I've not seen it anywhere in test. I'm wondering if this is a half finished PR/piece of work? From a design POV I'm not sure this new view really 'simplifies' the process that PM's go through

**Comment by frjo** [comment]
You can view/add/edit summaries in the React view but not in the standard view as of now. Easy to add if someone plans on using it.

**Comment by fourthletter** [comment]
> You can view/add/edit summaries in the React view but not in the standard view as of now. Easy to add if someone plans on using it.

Sounds like this issue was completed and could be closed.
